src/features/Logout/Logout.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+import Spinner from '@/components/Spinner';
+
+const Logout = () => {
+  useEffect(() => {
+    const logoutSession = async () => {
+      try {
+        const response = await fetch('/api/logout');
+        if (response.ok) {
+          window.location.href = '/login';
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    logoutSession();
+  }, []);
+
+  return <Spinner variant="large" />;
```

Comment:
Just a nitpick, but we could improve the visual of this. Now the spinner is just on the top-left. But yeah, maybe not top priority =)

Reply:
Yeah, also the spinner on the mentor page is in the left corner. I added an item to the backlog (https://trello.com/c/tsHqgTQF/741-mentor-page-logout-spinner-is-not-centered-or-padded), but maybe it would be quicker to just fix it. 😄

Reply:
Spinner now centered and margin added!